pkg/convert/pot_test.go

```diff
@@ -49,7 +51,7 @@ msgstr "Au revoir!"
 `),
 		},
 		{
-			name: "When description is multiline",
+			name: "description is multiline",
```

Comment:
I think it could be shortened even more:
```suggestion
			name: "multiline description",
```

and for all others test names too